### PR TITLE
[technical] Migrate Flutter Dribbble Nke to Flutter 2

### PR DESCRIPTION
**Pull request by M-A-D-A-R-A**
[#12](https://github.com/joshuadeguzman/flutter-examples/issues/12)

Please review the changes @joshuadeguzman

### CODE REVIEW

File: flutter_dribbble_nike_sb/lib/widgets/app_cta_button.dart [hunk 1]
```diff
@@ -9,9 +9,9 @@ class AppCtaButton extends StatefulWidget {
   final Function(bool isTapped) onCtaTap;
 
   const AppCtaButton({
-    Key key,
-    @required this.title,
-    this.onCtaTap,
+    Key? key,
+    /*required*/ required this.title,
```
Comment (joshuadeguzman) [inline flutter_dribbble_nike_sb/lib/widgets/app_cta_button.dart:13]: Might be a leftover hint from the migration tool?

File: flutter_dribbble_nike_sb/lib/widgets/app_cta_button.dart [hunk 2]
```diff
@@ -22,7 +22,7 @@ class AppCtaButton extends StatefulWidget {
 
 class _AppCtaButtonState extends State<AppCtaButton> {
   String get _title => widget.title;
-  Function get _onCtaTap => widget.onCtaTap;
+  Function/*!*/ get _onCtaTap => widget.onCtaTap;
```
Comment (joshuadeguzman) [inline flutter_dribbble_nike_sb/lib/widgets/app_cta_button.dart]: Similar to this one, kindly check the rest. Here's a reference that can help you: https://dart.dev/null-safety/migration-guide

**Review by joshuadeguzman — CHANGES_REQUESTED** [review]
Awesome, thank you @M-A-D-A-R-A! You might need to apply the migrations first to remove the hints coming from the migration tool.

**Comment by joshuadeguzman** [comment]
Hey @M-A-D-A-R-A, I would like to know if you have any updates on this. :D

**Comment by M-A-D-A-R-A** [comment]
> Hey @M-A-D-A-R-A, I would like to know if you have any updates on this. :D

Yeah sorry i was having my sem exams Ill looking it as soon as i can . Thanks!

**Pull request closed without merging**